### PR TITLE
Add agenda item for cranelift meeting on 11-29

### PR DESCRIPTION
**Pull request by jlb6740**

### CODE REVIEW

File: meetings/cranelift/2021/cranelift-11-29.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
     1. _Sumbit a PR to add your announcement here_
 1. Other agenda items
     1. _Sumbit a PR to add your item here_
-
+    Bug scrub - Maybe this belongs as a Wasmtime open but there are 369 Wasmtime issues dating back to 2016. Surely many are cranelift specific. Many are bugs likely in code that no longer exists. What's the best way to scrub these?
```
Comment (cfallin) [inline meetings/cranelift/2021/cranelift-11-29.md:15]: Tiny nit but could you prefix with `1.` to keep the Markdown formatting?

**Review by cfallin — APPROVED** [review]
Definitely a good time to talk about this, thanks!

File: meetings/cranelift/2021/cranelift-11-29.md (hunk 1 above)
Comment (bjorn3) [inline meetings/cranelift/2021/cranelift-11-29.md:15]: :+1: There are 9 search results for peepmatic: https://github.com/bytecodealliance/wasmtime/issues?q=is%3Aissue+is%3Aopen+peepmatic I think these can be closed too. There are also several wasi issues that I think should be moved to https://github.com/WebAssembly/WASI.

Comment (jlb6740) [reply meetings/cranelift/2021/cranelift-11-29.md]: Not a nit, thanks 👍

**Pull request merged**